### PR TITLE
Fixed problems and add some supports functions

### PR DESCRIPTION
**Pull request by leomaurodesenv**
It was edited some functions and added some support functions:

**Editions**
- remove std
- remove `stoi()`
- fixed `C4Bot::setting()`

**New**
- Code::Blocks IDE
- function `C4Bot::string2int()`
- function `C4Bot::get_player()`
- function `C4Bot::print()`
- Improvement in README.md and MIT License

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+C4Bot: connect four starter bot for the riddles.io platform
+Copyright (C) 2018, <https://github.com/afvanwoudenberg/c4bot>
+
+This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; either version 2 of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with this program; if not, write to the Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
```
Comment (kleisauke) [inline LICENSE]: This is not an MIT license as stated in the description of the pull request. And I don't think that @afvanwoudenberg wants a license linked to this starter bot. See: https://github.com/riddlesio/issues/issues/41#issue-257647806.

File: c4.cpp [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+/* c4.cpp
+ * Created by Aswin van Woudenberg <https://github.com/afvanwoudenberg>
+ * Collaboration of Leonardo Mauro <https://github.com/leomaurodesenv>
+ * link: https://github.com/afvanwoudenberg/c4bot
+ */
 
 #include "c4.h"
 
-std::ostream &operator<<(std::ostream& os, const Player &p) {
+using namespace std;
```
Comment (kleisauke) [inline c4.cpp]: Bad practice, please see: https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice.

File: c4bot.cpp [hunk 3]
```diff
@@ -1,84 +1,115 @@
...
+	string line;
+	while (getline(cin, line)) {
+		vector<string> command = split(line, ' ');
 		if (command[0] == "settings") {
 			setting(command[1], command[2]);
 		} else if (command[0] == "update" && command[1] == "game") {
 			update(command[2], command[3]);
 		} else if (command[0] == "action" && command[1] == "move") {
-			move(std::stoi(command[2]));
+			move(string2int(command[2]));
```
Comment (kleisauke) [inline c4bot.cpp]: What's wrong with `std::stoi`?

File: codeblocks/c4bot.cbp [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+		<Unit filename="../c4bot.cpp" />
+		<Unit filename="../c4bot.h" />
+		<Unit filename="../main.cpp" />
+		<Unit filename="tests.in" />
+		<Extensions>
+			<code_completion />
+			<debugger />
+		</Extensions>
+	</Project>
+</CodeBlocks_project_file>
```
Comment (kleisauke) [inline codeblocks/c4bot.cbp]: Why should these files be added? These files looks unnecessary for me and I'm sure there are better IDE's than `Code::Blocks`.

File: codeblocks/test_file.in [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+update game field .,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,1,.,.,.,.,1,.,0,.,.,.,.,1,.,0,.,.,.,1,0,0,0,1
+action move 10000
+update game round 7
+update game field .,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,0,.,1,.,.,.,.,1,.,0,.,.,.,1,1,.,0,.,.,.,1,0,0,0,1
+action move 10000
+update game round 8
+update game field .,.,.,.,.,.,.,.,.,.,0,.,.,.,.,.,.,0,.,1,.,.,.,.,1,.,0,.,.,.,1,1,.,0,.,.,1,1,0,0,0,1
+action move 10000
+update game round 9
+update game field .,.,.,0,.,.,.,.,.,.,0,.,.,.,.,.,.,0,.,1,.,.,.,.,1,.,0,.,.,1,1,1,.,0,.,.,1,1,0,0,0,1
```
Comment (kleisauke) [inline codeblocks/test_file.in]: You could use https://github.com/jmerle/ai-bot-workspace for testing purposes. I don't think this file belongs to this repo.

File: main.cpp [hunk 6]
```diff
@@ -1,12 +1,26 @@
...
+ * Collaboration of Leonardo Mauro <https://github.com/leomaurodesenv>
+ * link: https://github.com/afvanwoudenberg/c4bot
+ */
 
 #include "c4bot.h"
 
+#include <iostream>
+#include <sstream>
+#include <cstdlib>
+#include <cstdio>
```
Comment (kleisauke) [inline main.cpp]: Unnecessary includes.

File: LICENSE (hunk 1 above)
Comment (leomaurodesenv) [reply LICENSE]: Thanks a lot. I didnt see that!

File: c4bot.cpp (hunk 3 above)
Comment (leomaurodesenv) [reply c4bot.cpp]: Problems with users in Windows <=  g++ 4.6.x on MinGW

File: codeblocks/test_file.in (hunk 5 above)
Comment (leomaurodesenv) [reply codeblocks/test_file.in]: No really doesnt. Thanks for you help me.

**Comment by leomaurodesenv** [comment]
Solved! Thanks @kleisauke and @afvanwoudenberg .

**Pull request closed without merging**